src/components/molecules/ToolCard.tsx

```
@@ -0,0 +1,270 @@
+    return (
+        <div className="tool-card w-full">
+            {is_required_user_permission ?
+                <div>
+                    <div className="mb-3" style={{ overflowWrap: "anywhere" }}>
+                        {content}
+                    </div>
+                    <div>
+                        {cmds.filter(cmd => cmd.result == null).map(({ call_id, args, name, method, description }) => {
+                            return (
+                                <div className="cursor-default h-full py-3 px-6 border border-neutral-300 rounded-md shadow">
+                                    <h3 className="flex items-center justify-between text-xl font-semibold text-neutral-700 border-b pb-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)